benefit/data/sheets/CaseField/CaseField.json

```diff
@@ -421,6 +421,7 @@
   {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "ID": "ogdType", "Label": "OGD type", "FieldType": "Text", "SecurityClassification": "PUBLIC"},
   {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "ID": "caseManagementCategory", "Label": "Case management category", "FieldType": "DynamicList", "SecurityClassification": "PUBLIC"},
   {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "ID": "caseAccessCategory", "Label": "Case access category", "FieldType": "Text", "SecurityClassification": "PUBLIC"},
+  {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "ID": "CaseAccessCategory", "Label": "Case access category", "FieldType": "Text", "SecurityClassification": "PUBLIC"},
```

Comment:
If this is new we should add the current date just to remember that this was added recently